rundeck/config.sls

```diff
@@ -90,13 +90,23 @@ rundeck.login:
 {% if 'sshkey' in rundeck_settings %}
 {% for dir,dir_options in rundeck_settings.sshkey.items() %}
 
+rundeck.sshkey.{{ dir }}:
+    file.directory:
```

Comment:
Does this indention work?  I usually use two space indentation as my personal convention; feels more natural.

Reply:
Good catch @noelmcloughlin.  Two spaces are the [suggested syntax for YAML files](https://docs.saltstack.com/en/latest/topics/troubleshooting/yaml_idiosyncrasies.html) @sticky-note, so please modify this as well.

Reply:
I do two spaces indentations too, but not wanted to change this because whole formula is 4 spaces indentations. I can do a PR to change this in whole formula instead.

Reply:
So that explains why I didn't notice it!  Absolutely fine, let's stick to the current convention and that can be corrected in the future.